crypto/key_manager.go

```
@@ -23,13 +23,18 @@ type KeyManager interface {
 	// GetPublicKey returns the public key previously loaded. Potentially nil if the key
 	// manager requires initialization and this has not been done
 	GetPublicKey() crypto.PublicKey
+	// GetRawPublicKey returns the DER encoded public key bytes.
+	// This is needed for some applications that exchange or embed key hashes in structures.
+	// The result will be nil if a public key has not been loaded
```

Why not also return an error?